cross-chain/optimism/external/optimismGoerli/OptimismTokenBridge.json

```diff
@@ -0,0 +1,4 @@
+{
+  "address": "0x1111111111111111111111111111111111111111",
```

Comment:
I think the address will be `0xC7A204bDBFe983FCD8d8E61D02b475D4073fF97e` given this attestation transaction: https://goerli-optimism.etherscan.io/tx/0xdc2f6ee2237fe967a7b89a05ac2c72fb3762078f18754e1a5cc61232bdb9a1cd. I'll check with the Wormhole team, maybe it's just missing in the documentation.

Reply:
changed [ae18a6e](https://github.com/keep-network/tbtc-v2/pull/585/commits/ae18a6e38e3778eeb5899e7af3cde1eaa920b23f)